dev-infrastructure/modules/logs/kusto/tables/backendContainerLogs.kql

```diff
@@ -0,0 +1,56 @@
+.create-merge table backendContainerLogs (
+  timestamp: datetime,
+  log: dynamic,
+  environment: string,
+  cluster: string,
+  namespace: string,
+  containerName: string,
+  containerImage: string,
+  host: string,
+  labels: string,
+  podName: string,
+  region: string,
+  level: string,
+  operation: string,
+  operationId: string,
+  resourceId: string,
+  internalId: string,
+  clusterId: string,
+  clientRequestId: string,
+  correlationRequestId: string,
+  cloudErrorCode: string,
+  cloudErrorMessage: string,
+  subscriptionId: string,
+  tag: string,
+  unmatched: dynamic
```

Comment:
Do you have any idea how we might catch new log statements being added to the frontend that either typo a field, use some other field (clusterID vs clusterId), etc?

Reply:
No, I think this requires manual rigor